### ISSUE TITLE
User's Guide table missing: Required Files for Different Configurations and Simulation Types

### ISSUE THREAD

**Issue by samsrabin** [ESCOMP/CTSM#2224]
The table that's supposed to be at the bottom of [this page](https://escomp.github.io/ctsm-docs/versions/master/html/users_guide/adding-new-resolutions/Adding-or-Changing-Default-Filenames.html) is missing. It's supposed to be Table 3-1 [here](https://www2.cesm.ucar.edu/models/cesm1.1/clm/models/lnd/clm/doc/UsersGuide/x8753.html).

**Comment by samsrabin** [comment]
@slevis-lmwg This is in Sect. 1.4, which according to #3883 is your bailiwick. Can I assign this issue to you?

**Comment by slevis-lmwg** [comment]
The Table 3-1 link above requires credentials other than CIT, so I gave up on that. The closest I got to relevant information (though not a table) is some text in https://docs.cesm.ucar.edu/models/cesm1/user-guide/1.1/x351.html#download_ccsm_inputdata:

> As part of the process of generating the CESM executable, the utility, [check_input_data](https://docs.cesm.ucar.edu/models/cesm1/user-guide/1.1/x1060.html#input_data_server) is called, and it attempts to locate all required input data for the case based upon file lists generated by components. If the required data is not found on local disk in $DIN_LOC_ROOT, then the data will be downloaded automatically by the scripts or it can be downloaded by the user by invoking check_input_data with the -export command argument. If you want to download the input data manually you should do it before you build CESM.
>
> It is possible for users to download the data using svn subcommands directly, but use of the check_input_data script is highly recommended to ensure that only the required datasets are downloaded. Again, users are STRONGLY DISCOURAGED from downloading the entire input dataset from the repository due to the size.

I am considering whether a table of input datasets for every conceivable configuration makes sense from a sane maintenance perspective. In its place it seems more manageable to use a variation of the above paragraphs that direct users to determine the input data relevant to their configuration.

Actually some reuse of the info starting here
https://docs.cesm.ucar.edu/models/cesm1/user-guide/1.1/c1001.html
and then following the "required data" links.

**Comment by samsrabin** [comment]
I agree, @slevis-lmwg—this kind of listing would be way more trouble than it's worth. The build-namelist script will tell the user pretty quickly if they're missing a file. If you agree, this issue can be resolved by deleting section 1.4.3.1.

**Comment by slevis-lmwg** [comment]
Thanks @samsrabin 
I like that idea and will open the PR today.
